tempCommit/PrintReads.wdl

```diff
@@ -0,0 +1,297 @@
+    ~{true='--add_output_vcf_command_line ' false='' defined(add_output_vcf_command_line)}~{sep=' --add_output_vcf_command_line ' add_output_vcf_command_line} \
+    ~{true='--create_output_bam_index ' false='' defined(create_output_bam_index)}~{sep=' --create_output_bam_index ' create_output_bam_index} \
+    ~{true='--create_output_bam_md5 ' false='' defined(create_output_bam_md5)}~{sep=' --create_output_bam_md5 ' create_output_bam_md5} \
+    ~{true='--create_output_variant_index ' false='' defined(create_output_variant_index)}~{sep=' --create_output_variant_index ' create_output_variant_index} \
+    ~{true='--create_output_variant_md5 ' false='' defined(create_output_variant_md5)}~{sep=' --create_output_variant_md5 ' create_output_variant_md5} \
+    ~{true='--disable_read_filter ' false='' defined(disable_read_filter)}~{sep=' --disable_read_filter ' disable_read_filter} \
+    ~{true='--disable_tool_default_read_filters ' false='' defined(disable_tool_default_read_filters)}~{sep=' --disable_tool_default_read_filters ' disable_tool_default_read_filters} \
+    ~{true='--exclude_intervals ' false='' defined(exclude_intervals)}~{sep=' --exclude_intervals ' exclude_intervals} \
+    ~{true='--gatk_config_file ' false='' defined(gatk_config_file)}~{sep=' --gatk_config_file ' gatk_config_file} \
+    ~{true='--interval_exclusion_padding ' false='' defined(interval_exclusion_padding)}~{sep=' --interval_exclusion_padding ' interval_exclusion_padding} \
+    ~{true='--interval_padding ' false='' defined(interval_padding)}~{sep=' --interval_padding ' interval_padding} \
+    ~{true='--interval_set_rule ' false='' defined(interval_set_rule)}~{sep=' --interval_set_rule ' interval_set_rule} \
+    ~{true='--lenient ' false='' defined(lenient)}~{sep=' --lenient ' lenient} \
+    ~{true='--QUIET ' false='' defined(QUIET)}~{sep=' --QUIET ' QUIET} \
+    ~{true='--read_filter ' false='' defined(read_filter)}~{sep=' --read_filter ' read_filter} \
+    ~{true='--read_index ' false='' defined(read_index)}~{sep=' --read_index ' read_index} \
+    ~{true='--read_validation_stringency ' false='' defined(read_validation_stringency)}~{sep=' --read_validation_stringency ' read_validation_stringency} \
+    ~{true='--seconds_between_progress_updates ' false='' defined(seconds_between_progress_updates)}~{sep=' --seconds_between_progress_updates ' seconds_between_progress_updates} \
+    ~{true='--sequence_dictionary ' false='' defined(sequence_dictionary)}~{sep=' --sequence_dictionary ' sequence_dictionary} \
+    ~{true='--tmp_dir ' false='' defined(tmp_dir)}~{sep=' --tmp_dir ' tmp_dir} \
+    ~{true='--use_jdk_deflater ' false='' defined(use_jdk_deflater)}~{sep=' --use_jdk_deflater ' use_jdk_deflater} \
+    ~{true='--use_jdk_inflater ' false='' defined(use_jdk_inflater)}~{sep=' --use_jdk_inflater ' use_jdk_inflater} \
+    ~{true='--verbosity ' false='' defined(verbosity)}~{sep=' --verbosity ' verbosity} \
```

Comment:
Do we need another newline here because of the trailing backslash?

Reply:
Oh, yeah, done.